cluster/saltbase/salt/kube-dns/transforms2sed.sed

```diff
@@ -1,4 +1,5 @@
 s/__PILLAR__DNS__SERVER__/$DNS_SERVER_IP/g
 s/__PILLAR__DNS__REPLICAS__/$DNS_REPLICAS/g
 s/__PILLAR__DNS__DOMAIN__/$DNS_DOMAIN/g
+s/__PILLAR__FEDERATIONS__DOMAIN__MAP__/$FEDERATIONS_DOMAIN_MAP/g
```

Comment:
Might be better to blank it out with a `s/xxxxxxxxx//g' depending on the decision for skydns-rc.yaml.sed.